web/templatetags/markdown_filters.py

```
@@ -1,11 +1,52 @@
+        protocols=ALLOWED_PROTOCOLS,
+        strip=True
+    )
+
 
 @register.filter
 def markdown(text):
-    """Convert markdown text to HTML."""
-    return mark_safe(markdownify(text))
+    """Convert markdown text to sanitized HTML."""
+    clean_html = markdownify_sanitized(text)
+    return mark_safe(clean_html)
```

Ruff (0.15.6)

[error] Use of `mark_safe` may expose cross-site scripting vulnerabilities

(S308)